docs/test/reference-qa.md

```diff
@@ -560,6 +560,39 @@ For more details, see [Add findings to existing bugs with exploratory testing](a
 2. Search for 'Use Blink's zoom for device scale factor'
 3. Change it to **Disabled**
 
+## FAQ on New Test Plans Page
+
+### Q: How can I assign configurations to my test cases?
+**A:** You can assign configurations in two ways: 
+
+a. To assign configuration to whole suite, go to test suite tree, select the test suite and open the in-context menu bar to assign configurations.
+
+b. To assign configuration to test case(s), navigate to test plan > test suite and click on the in-context menu bar of the test case.
+![FAQ1.2](_img/FAQ1.2.png)
+
+### Q: How can I see the configurations assigned to my test cases?
+**A:** Once you assign the configurations to the test cases in Define tab, the corresponding test points are created in the Execute tab. Navigate to Execute tab and see the configurations assigned to all the test points.
+![FAQ2](_img/FAQ2.png)
+
+### Q: How can I edit the test case?
+**A:** If you have Basic License, you can edit the test case by navigating to test plan > test suite > Execute  > Test Point and clicking on In-context menu bar. If you have Basic + Test Plans License, in addition to editing in Execute tab, you can also navigate to Define tab and double click on test case to edit it.
```

Comment:
'license' -> 'access level' in both these stmts.

Reply:
done